packages/boba/gas-price-oracle/src/service.ts

```diff
@@ -106,6 +110,30 @@ export class GasPriceOracleService extends BaseService<GasPriceOracleOptions> {
       address: this.state.Lib_AddressManager.address,
     })
 
+    this.logger.info('Connecting to Proxy__L1StandardBridge...')
+    const Proxy__L1StandardBridgeAddress =
+      await this.state.Lib_AddressManager.getAddress('Proxy__L1StandardBridge')
+    this.state.Proxy__L1LiquidityPool = new Contract(
```

Comment:
needs change to 'this.state.Proxy__L1StandardBridge'

Reply:
Yes! It was fixed.